src/middlewared/middlewared/etc_files/initiators.allow.mako

```diff
@@ -1,7 +1,18 @@
 <%
+    import socket
     base_name = middleware.call_sync('iscsi.global.config')['basename']
     targets = middleware.call_sync('iscsi.target.query', [['auth_networks', '!=', []]])
+
+    def parse_auth(auth):
+        s = auth.split('/')
+        try:
+            if len(s) == 2 and s[1] == '32':
+                socket.inet_pton(socket.AF_INET, s[0])
```

Comment:
What about IPv6 addresses? I know we can't test this (still waiting on some internal infrastructure changes).

Reply:
Also, I believe using the builtin `ipaddress.ip_interface()` method would be a bit easier to use here.

Reply:
Something like
```
try:
    ipobj = ipaddress.ip_interface('address-here')
except ValueError:
    log-something-about-invalid-ipaddress
else:
    return str(ipobj.ip)

Reply:
Agreed, it doesn't handle IPv6 properly yet, but the `inet_pton` check is there so that we can add another check to the `parse_auth` function to do so (e.g. another try..except) when we get it plumbed thru.

We can add code now, but as you say can't test yet.